Project.toml

```diff
@@ -1,6 +1,9 @@
 name = "ReinforcementLearning"
 uuid = "158674fc-8238-5cab-b5ba-03dfc80d1318"
-authors = ["Johanni Brea <jbrea@users.noreply.github.com>", "Jun Tian <tianjun.cpp@gmail.com>"]
+authors = [
+    "Johanni Brea <jbrea@users.noreply.github.com>",
```

Comment:
Please revert the changes to this file, at the moment the umbrella package isn't being shipped with the latest versions of the dependencies.

Reply:
This causes issues when I try to compile the umbrella package to fail to pre-compile. 

If I try to dev the sub-packages I get 

```
(ReinforcementLearning) pkg> dev src/ReinforcementLearningZoo src/ReinforcementLearningEnvironments src/ReinforcementLearningCore src/ReinforcementLearningBase
   Resolving package versions...
ERROR: empty intersection between ReinforcementLearningZoo@0.10.1 and project compatibility 0.6
```

What do you suggest?